packages/server/lib/config.ts

```diff
@@ -65,8 +64,13 @@ const convertRelativeToAbsolutePaths = (projectRoot, obj, defaults = {}) => {
 }
 
 const validateNoBreakingConfig = (cfg) => {
-  return _.each(breakingOptions, ({ name, errorKey, newName, isWarning }) => {
-    if (_.has(cfg, name)) {
+  breakingOptions.forEach(({ name, errorKey, newName, isWarning, value }) => {
```

Comment:
Updated this to allow me to flex the warning message on the value of the config option